### DISCUSSION TITLE
Help me removing the player indicator

### DISCUSSION THREAD

**Discussion by genjutsus** [wxyz-abcd/node-haxball#68] [General]
![image](https://github.com/wxyz-abcd/node-haxball/assets/154602378/0b094bd9-e238-456e-a60f-f2c85fb68f5d)

This white line. Please. It's just so frustrating. I have zero coding knowledge but can set a Snippet in Developer Tools.

**Comment by wxyz-abcd** [comment]
You can use the "currentPlayerDistinction" property in SandboxRenderer. Reference: https://github.com/wxyz-abcd/node-haxball/blob/2802e79c30711975d51885cf6b32ad13202b13ba/examples/renderers/sandboxRenderer.js#L113
